src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/RequestSql/SqlRequestType.php

```diff
@@ -26,24 +26,52 @@
 
 namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\RequestSql;
 
+use PrestaShopBundle\Translation\TranslatorAwareTrait;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
 
 /**
  * Class RequestSqlType defines RequestSql entity form type.
  */
 class SqlRequestType extends AbstractType
 {
+    use TranslatorAwareTrait;
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name', TextType::class)
-            ->add('sql', TextareaType::class)
+            ->add('name', TextType::class, [
+                'constraints' => [
+                    new NotBlank([
+                        'message' => $this->trans(
+                            'The %s field is required.',
+                            [
+                                sprintf('"%s"', $this->trans('SQL query name', [], 'Admin.Advparameters.Feature')),
```

Comment:
sprintf is useless here.
```php
$this->trans(
    'The "%s" field is required.',
    [
        $this->trans('SQL query name', [], 'Admin.Advparameters.Feature'),
    ],
    'Admin.Notifications.Error'
),
```

Same below

Reply:
we would lose translation if we did that. :/